.buildkite/longruns/pipeline.yml

```diff
@@ -283,105 +294,13 @@ steps:
         agents:
           slurm_gpus: 1
           slurm_mem: 16GB
+        soft_fail: true
 
-  - group: "Other AMIP targets"
-
-    steps:
-
-      # DYAMOND AMIP: 1 day (convection resolving)
-      - label: "MPI AMIP SUPERFINE: dyamond_target"
```

Comment:
are we still interested in DYAMOND, or not anymore?

Reply:
We are, but the gpu version (`gpu_dyamond_target`) should be sufficient here.